go/billing/tasks.py

```diff
@@ -547,7 +548,9 @@ def create_low_credit_notification(
             'reference': notification.id,
         })
 
-    email = EmailMessage(subject, message, email_from, [email_to])
+    # remove duplicate emails from the recipient list
+    recipient_emails = list(set([email_to, support_email]))
+    email = EmailMessage(subject, message, email_from, recipient_emails)
```

Comment:
done :)
The test doesn't need to be fixed as it checks recipients():
"recipients() returns a list of all the recipients of the message, whether they’re recorded in the to, cc or bcc attributes"